packages/components/src/private-apis.ts

```diff
@@ -61,4 +62,5 @@ lock( privateApis, {
 	DropdownSubMenuTriggerV2,
 	ProgressBar,
 	Theme,
+	Tabs,
```

Comment:
We're intentionally keeping the `Tabs.` namespace because we're aiming to access the components with `Tabs.TabList` and `Tabs.Tab` right?

Reply:
Correct!